src/cascadia/TerminalApp/CommandPalette.cpp

```diff
@@ -28,7 +28,8 @@ namespace winrt::TerminalApp::implementation
         _nestedActionStack = winrt::single_threaded_vector<Command>();
         _currentNestedCommands = winrt::single_threaded_vector<Command>();
         _allCommands = winrt::single_threaded_vector<Command>();
-        _allTabActions = winrt::single_threaded_vector<Command>();
+        _inOrderTabActions = winrt::single_threaded_vector<Command>();
+        _mruTabActions = winrt::single_threaded_vector<Command>();
```

Comment:
Oof so we always store both even though we can only technically use one of the modes at a time? Is that so on the fly switching works and remembers the history?